src/main/java/com/panel/app/service/CartServiceImpl.java

```
@@ -0,0 +1,108 @@
+package com.panel.app.service;
+
+import com.fruits.ecommerce.model.*;
+import com.panel.app.entity.Cart;
+import com.panel.app.entity.CartItem;
+import com.panel.app.entity.Product;
+import com.panel.app.entity.User;
+import com.panel.app.repo.CartRepo;
+import com.panel.app.repo.ProductRepo;
+import com.panel.app.repo.UserRepo;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CartServiceImpl implements CartService{
+
+    @Autowired
+    private UserRepo userRepo;
+
+    @Autowired
+    private ProductRepo productRepo;
+
+    @Autowired
+    private CartRepo cartRepo;
+
+
+
+    @Override
+    public String addToCart(long userId, long productId) {
```

Method should return void instead, and throw exception when user is not found